sunpy/map/tests/test_mapsequence.py

```diff
@@ -196,3 +198,23 @@ def test_norm_animator(hmi_test_map):
 def test_map_sequence_plot(aia171_test_map, hmi_test_map):
     seq = sunpy.map.Map([aia171_test_map, hmi_test_map], sequence=True)
     seq.plot()
+
+
+def test_save(aia171_test_map, hmi_test_map):
+    """Tests the map save function"""
+    seq = sunpy.map.Map([aia171_test_map, hmi_test_map], sequence=True)
+    afilename = tempfile.NamedTemporaryFile(suffix='fits').name
+    seq.save(afilename, filetype='fits', overwrite=True)
+    test_seq = sunpy.map.Map(afilename+"_0", afilename+"_1", sequence=True)
```

Comment:
So the saved files don't have the fits extension?

Reply:
Nope, here it just adds fits to the end of the filename, not really an extension. Took this from `GenericMap` and updated it, should I change that as well?

Reply:
You mean the Map load adds the "fits" extension on load?

Reply:
Oh wait, I got confused, it is of `FITS` type but has no extension such as `filename.fits`. 
This is provided the `filetype` attribute is set = `fits`.

Reply:
So the saved file has no "fits" extension in the file name?

Reply:
No, they're of type `FITS` but are saved with just the filename

Reply:
Ah I see. I think changing that is outside of scope for this PR.